### PR TITLE
Show warning message for users pending approval

### PR DESCRIPTION
**Pull request by Yoshani**
> This PR introduces a better approach for fixing issue [1] by keeping the initial behaviour for users created successfully and for users pending approval, redirect to the user list page and show a warning message.

### Related Issues
- [1] Issue https://github.com/wso2/product-is/issues/14679
- [2] Issue https://github.com/wso2/product-is/issues/16027

### Related PRs
- Related PR https://github.com/wso2/identity-apps/pull/3927

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by chamathns — DISMISSED** [review]

**Comment by brionmario** [comment]
Ignore the ESLint check.
It's happening due to this change (https://github.com/wso2/identity-apps/pull/3933/files)

@pavinduLakshan Lets fix the modules with warning tolerance `0` or revert this PR until we properly fix.

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -10330,6 +10330,20 @@ export const console: ConsoleNS = {
                             message: "User added successfully"
                         }
                     },
+                    addUserPendingApproval: {
+                        error: {
+                            description: "{{description}}",
```
Comment (Achintha444) [inline modules/i18n/src/translations/en-US/portals/console.ts:10335]: Shall we add a description here.

File: apps/console/src/features/users/components/wizard/add-user-wizard.tsx [hunk 2]
```diff
@@ -469,22 +469,35 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
...
+                    dispatch(addAlert({
+                        description: t(
+                            "console:manage.features.users.notifications.addUserPendingApproval.success.description"
+                        ),
+                        level: AlertLevels.WARNING,
+                        message: t(
+                            "console:manage.features.users.notifications.addUserPendingApproval.success.message"
+                        )
+                    }));
+                    history.push(AppConstants.getPaths().get("USERS"));
```
Comment (Achintha444) [inline apps/console/src/features/users/components/wizard/add-user-wizard.tsx:482]: Isn't is possible to remove this line ?

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 1 above)
Comment (Yoshani) [reply modules/i18n/src/translations/en-US/portals/console.ts:10335]: All the other notification types have it this way. So not changing it.

**Review by Achintha444 — APPROVED** [review]

**Review by savindi7 — APPROVED** [review]

**Pull request merged**